save-orchestrator/src/main/resources/data.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE IF EXISTS agent_status;
```

Comment:
as we have decided - these scripts should be added to common gradle liquibase script